eZ/Publish/Core/FieldType/Tests/RichText/Converter/Xslt/_fixtures/xhtml5/edit/027-linkedEmbed.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <section xmlns="http://ez.no/namespaces/ezpublish5/xhtml5/edit">
-  <ezembed href="ezcontent://106" id="embed-id-1" data-ezview="embed" class="embed-class" data-ezalign="left">
-    <ezlink href="ezurl://95#fragment1" target="_blank" id="link-id-1" title="Link title" class="link-class"/>
-    <ezconfig>
-      <ezvalue key="size">medium</ezvalue>
-      <ezvalue key="offset">10</ezvalue>
-      <ezvalue key="limit">5</ezvalue>
-    </ezconfig>
-  </ezembed>
-  <ezembed href="ezlocation://601" id="embed-id-2" data-ezview="line" class="embedClass2" data-ezalign="right">
-    <ezlink href="ezlocation://202" class="linkClass2"/>
-  </ezembed>
+  <div data-ezelement="ezembed" href="ezcontent://106" id="embed-id-1" data-ezview="embed" class="embed-class" data-ezalign="left">
+    <a data-ezelement="ezlink" href="ezurl://95#fragment1" target="_blank" id="link-id-1" title="Link title" class="link-class"/>
```

Comment:
it's a weird markup, to improve it a bit, I would rather use a `link` tag here. According to http://www.w3.org/html/wg/drafts/html/master/semantics.html#the-link-element, it should have an `itemprop` attribute, I suggest to set its value to `url` and that also means the `div` has to have an `itemscope` attribute as well.